third-party/rspamd/src/test/resources/rspamd-config/actions.conf

```diff
@@ -1,29 +0,0 @@
-# Actions settings
```

Comment:
this file is unnecessary?
Don't forget we haven't any spam.eml sample file with scores greater than 15